vignettes/articles/input_params.csv

```diff
@@ -0,0 +1,223 @@
+section,parameter,biome-specific,time-variant,required,default,units,description
```

Comment:
Looking at the generated html tables the I see some unit entries`W yr m<96>2 C Tg<96>1` when I open the csv file there are `W yr mñ2 C Tgñ1` entries could you make sure that the entires that the units are formatted properly?